packages/turf-transform-scale/index.js

```diff
@@ -46,39 +46,33 @@ module.exports = function (geojson, factor, origin, mutate) {
     // Scale each Feature separately
     if (geojson.type === 'FeatureCollection' && !originIsPoint) {
         featureEach(geojson, function (feature, index) {
-            geojson.features[index] = scale(feature, factor, origin, mutate);
+            geojson.features[index] = scale(feature, factor, origin);
         });
         return geojson;
     }
     // Scale Feature/Geometry
-    return scale(geojson, factor, origin, mutate);
+    return scale(geojson, factor, origin);
 };
 
 /**
  * Scale Feature/Geometry
  *
  * @private
- * @param {Feature|Geometry} geojson GeoJSON Feature/Geometry
+ * @param {GeoJSON} geojson GeoJSON Feature/Geometry
```

Comment:
GeoJSON means it handles FeatureCollection & GeometryCollection.

This doesn't?